cloud-stream-manual-ack/README.md

```diff
@@ -0,0 +1,131 @@
+# Manual Message Acknowledgement
+
+In general, a received message is automatically acknowledged. This is true whether the message is consumed or results in an error. To manage the acknowledgement manually, the message handlers can disable auto-acknowledgment and manually invoke the acknowledgement callback. 
```

Comment:
same language that needed to be changed in blog, maybe it just needs to be copied over?

Reply:
make other changes to this README if language changed in the blog as well.

Reply:
Copied over the blog text into README (portions of it as applicable)